src/contexts/MapStyleContext.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext, useState } from 'react';
```

Comment:
compare this context with the two contexts we already have. Use the same structure and naming convention